README.md

```diff
@@ -192,3 +192,22 @@ bin/console doctrine:migrations:migrate
 ```
 docker-compose exec frontend bash -c 'npm run build'
 ```
+
+## Translations
+The React front end used (formatJS)[https://formatjs.io/] to handle translations.
+
+To extract new strings in json.
+```sh
+docker-compose exec frontend bash -c "npm run extract -- 'assets/**/*.js*' --out-file public/lang/en.json --id-interpolation-pattern '[sha512:contenthash:base64:6]'"
+```
+Make copy of this file into new JSON file and with language code and change the
+strings into your language and compile then json file with this command
+substitute `<LANGCODE>` with your language code.
+```sh
+docker-compose exec frontend bash -c "npm run compile -- public/lang/<LANGCODE>.json --ast --out-file public/lang/<LANGCODE>-comp.json"
+```
+
+Now to the not so pretty part edit `assets/js/app.js` and locate the
```

Comment:
"Now to the not so pretty part" => "To add the new language, "

Reply:
Fixed